.github/workflows/charts-release-action.yml

```diff
@@ -41,8 +51,9 @@ jobs:
           git fetch origin
           if git ls-remote --heads origin $branch_name | grep $branch_name; then
             echo "Branch $branch_name already exists. Skipping creation."
+            git checkout $branch_name
           else
-            git branch $branch_name
+            git checkout -b $branch_name
```

Comment:
What is the branch that git is running in when it decides to create a new branch?

Reply:
It will always be the default branch (main) unless otherwise specified.

Reply:
Is that what you want. What if we need to patch a release that's a few revisions back? Don't you want to checkout from the release tag instead? Main could have incompatible changes right.

Reply:
For a patch, the branch should already exist with the changes.

Reply:
OK